pages/plugins/_tutorial_readme.html

```diff
@@ -0,0 +1,36 @@
+<pre><code># File Counter Buildkite Plugin
```

Comment:
Does this formatting need fixing? Or did that already happen in another PR?
![image](https://user-images.githubusercontent.com/95874/77087686-03201a80-69c9-11ea-9c62-c3d12e8bdafe.png)

Reply:
Ah, it's fixed in master (and I think in another PR). I'll triple check the rendering in the backend rendered version too (it's what I was mostly using), as that's what will go live.